### ISSUE TITLE
configure error

### ISSUE THREAD

**Issue by glc412430525** [jbeezley/convert_geotiff#13]
I've installed the dependency library (libtiff and geotiff) and gave the path of CFLAGS and LDFLAGS. But there are still error when configuring. Kindly help me resolve the issue.
Looking forward to any kind of help.
![image](https://github.com/openwfm/convert_geotiff/assets/49713550/f7659092-b0b3-491c-bd8b-eebf4ea05863)
I guess maybe my dependency library installation was successful, but incomplete.

**Comment by Cindy-86** [comment]
Please help，did you solve this problem？ i met it too，

**Comment by glc412430525** [comment]
> Please help，did you solve this problem？ i met it too，
I ended up using online installation.

**Comment by Cindy-86** [comment]
could you share your steps about “online installation”

**Comment by glc412430525** [comment]
Sorry. It was used to install the library online, but I forgot.

**Comment by glc412430525** [comment]
Try this.
1. conda install -c conda-forge gdal -y
2. pip install geotifflib

**Comment by Cindy-86** [comment]
OK thank you so much



---Original---
From: ***@***.***&gt;
Date: Mon, Nov 11, 2024 15:37 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [openwfm/convert_geotiff] configure error (Issue #13)




 
Try this.
 conda install -c conda-forge gdal -y

**Comment by Cindy-86** [comment]
you mean，don’t use “convert_geotiff”？directly “gdal”？



---Original---
From: ***@***.***&gt;
Date: Mon, Nov 11, 2024 15:37 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [openwfm/convert_geotiff] configure error (Issue #13)




 
Try this.
 conda install -c conda-forge gdal -y

**Comment by glc412430525** [comment]
No, install these two libraries first (conda install -c conda-forge gdal -y
pip install geotifflib), and then use the previous steps to install this one (convert_geotiff).

**Comment by Cindy-86** [comment]
ok i will have a try
